test/bdd/common_steps.go

```diff
@@ -632,12 +633,16 @@ func (d *CommonSteps) httpPost(url, data, contentType string) error {
 
 	data = resolved.(string)
 
+	println(fmt.Sprintf("!!!DEREK in httpPost step method, url=[%s], data=[%s], contentType=[%s]", url, data, contentType))
+
 	resp, err := d.doHTTPPost(url, []byte(data), contentType)
 	if err != nil {
 		return err
 	}
 
 	if resp.StatusCode != http.StatusOK {
+		println(fmt.Sprintf("!!!DEREK in httpPost step method, did not get status OK, url=[%s], data=[%s], contentType=[%s]", url, data, contentType))
```

Comment:
remove this msg

Reply:
Oops, I didn't mean to commit this. I removed it.